scripts/data_analysis/generate_schema_diagram.py

```diff
@@ -0,0 +1,73 @@
+import json
+import requests
+import yaml
+import pandas as pd
+import os
+import sys
+
+from graphviz import Digraph
+from constants import common
+
+OUTPUT_FILE = common.SCHEMA_PARENT_CHILD_FILE
+DIAGRAM_FILE = common.SCHEMA_DIAGRAM
+
+response_dict = common.extract_data(common.URL)
+
+def main():
+    common.check_directory(DIAGRAM_FILE)
+    data_df = pd.DataFrame()
+
+    if len(sys.argv) == 1: #No arguments passed in - full schema diagram
```

Comment:
```suggestion
    is_full_schema_diagram = len(sys.argv) == 1
    
    if is_full_schema_diagram:
```

Would rather us do something like this so we can remove the comment and is moar readable imo.

Reply:
Yes - was trying to think of the best way to do this, I like this!